src/sentry/static/sentry/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -258,11 +269,11 @@ class TeamMembers extends React.Component {
     );
   };
 
-  removeButton = member => (
+  removeButton = (member: Member) => (
     <Button
       size="small"
       icon={<IconSubtract size="xs" isCircled />}
-      onClick={this.removeMember.bind(this, member)}
+      onClick={() => this.removeMember(member)}
```

Comment:
if you update the removeMember function, here could be like 

`onClick={this.removeMember(member)}`

it's just a suggestion